wagtail/contrib/search_promotions/admin_urls.py

```diff
@@ -1,6 +1,7 @@
 from django.urls import path
 
 from wagtail.contrib.search_promotions import views
+from wagtail.contrib.search_promotions.search_terms import SearchTermsReportView
```

Comment:
Would be better for this to live at `wagtail.contrib.search_promotions.views.reports`.